content/bn/edge-computing.md

```diff
@@ -0,0 +1,17 @@
+---
+title: এজ কম্পিউটিং (Edge Computing)
+status: Completed
+category: ধারণা
+---
+
+## এটা কি
+
+এজ কম্পিউটিং হল একটি [বিতরণ সিস্টেম](/bn/distributed-systems/) পদ্ধতি যা প্রাথমিক ডেটা সেন্টার থেকে ডেটা উৎসে কিছু স্টোরেজ এবং কম্পিউটিং ক্ষমতা স্থানান্তর করে। সংগৃহীত ডেটা স্থানীয়ভাবে গণনা করা হয় (যেমন, একটি কারখানার মেঝেতে,একটি দোকানে বা একটি শহর জুড়ে) প্রক্রিয়াকরণ (processing) এবং বিশ্লেষণের জন্য কেন্দ্রীভূত ডেটা সেন্টারে পাঠানোর পরিবর্তে। এই স্থানীয় প্রক্রিয়াকরণ ইউনিট বা ডিভাইসগুলি সিস্টেমের প্রান্তের (edge) প্রতিনিধিত্ব করে,যেখানে ডেটা সেন্টার হল এর কেন্দ্র। প্রান্তে গণনা করা আউটপুট পরবর্তী প্রক্রিয়াকরণের জন্য প্রাথমিক ডেটা সেন্টারে ফেরত পাঠানো হয়। এজ কম্পিউটিংয়ের উদাহরণগুলির মধ্যে রয়েছে হাতের কব্জির গ্যাজেট বা কম্পিউটার যা ট্র্যাফিক প্রবাহ বিশ্লেষণ করে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+গত এক দশকে,আমরা এজ ডিভাইসের (যেমন,মোবাইল ফোন,স্মার্ট ঘড়ি বা সেন্সর) ক্রমবর্ধমান পরিমাণ দেখেছি। কিছু ক্ষেত্রে,রিয়েল-টাইম ডেটা প্রসেসিং শুধুমাত্র রুচিকর নয় গুরুত্বপূর্ণও। যেমন স্ব-চালিত গাড়ি। এখন কল্পনা করা যাক একটি গাড়ির সেন্সর থেকে ডেটা গাড়িতে ফেরত পাঠানোর আগে প্রক্রিয়াকরণের জন্য একটি ডেটা সেন্টারে স্থানান্তর করতে হবে যাতে এটি যথাযথভাবে প্রতিক্রিয়া করতে পারে। অন্তর্নিহিত নেটওয়ার্ক লেটেন্সি মারাত্মক হতে পারে। যদিও এটি একটি চরম উদাহরণ,বেশিরভাগ ব্যবহারকারী স্মার্ট ডিভাইস ব্যবহার করতে চান না কারণ এটি তাত্ক্ষণিক প্রতিক্রিয়া প্রদান করতে অক্ষম।
+
+## এটা কিভাবে সাহায্য করে
+
+উপরের বর্ণনা হিসাবে,এজ ডিভাইসগুলিকে প্রয়োজনে ব্যাবহার করতে হলে,ডিভাইসগুলিকে অবশ্যই অন্তত আংশিকভাবে  প্রক্রিয়াকরণ এবং স্থানীয় বিশ্লেষণ করতে হবে ব্যবহারকারীদের কাছে রিয়েল-টাইম প্রতিক্রিয়া প্রদানের জন্য। ডেটা সেন্টার থেকে যেখানে ডেটা তৈরি হয় সেখানে কিছু স্টোরেজ স্থানান্তর এবং সংস্থান প্রক্রিয়াকরণ করে এটি অর্জন করা হয়:এজ ডিভাইস। প্রক্রিয়াজাত এবং অপ্রক্রিয়াজাত ডেটা পরবর্তীতে আরও প্রক্রিয়াকরণ এবং স্টোরেজের জন্য ডেটা সেন্টারে পাঠানো হয়। সংক্ষেপে,দক্ষতা এবং গতি এজ কম্পিউটিং এর প্রাথমিক উপাদান।
```

Comment:
Can you please make a more suitable sentence instead of this? "ডেটা সেন্টার থেকে যেখানে ডেটা তৈরি হয় সেখানে কিছু স্টোরেজ স্থানান্তর এবং সংস্থান প্রক্রিয়াকরণ করে এটি অর্জন করা হয়:এজ ডিভাইস"